src/data/nat/prime.lean

```diff
@@ -131,6 +131,9 @@ section min_fac
   (nat.sub_lt_sub_right_iff $ le_sqrt.2 $ le_of_not_gt h).2 $
   nat.lt_add_of_pos_right dec_trivial
 
+  /-- If `n < k * k`, then `min_fac_aux n k = n`, if `k | n`, then `min_fac_aux n k = k`.
+    Otherwise, `min_fac_aux n (k + 2) = n` using well-founded recursion.
```

Comment:
Maybe I misunderstood the function definition, but did you mean this instead?
```suggestion
    Otherwise, `min_fac_aux n = min_fac_aux (k + 2)` using well-founded recursion.
```